src/Documents/DocumentStore.ts

```diff
@@ -0,0 +1,60 @@
+import {IDocument} from './IDocument';
+import {IDocumentStore} from './IDocumentStore';
+import {IDocumentSession} from "./Session/IDocumentSession";
+import {DocumentSession} from "./Session/DocumentSession";
+import {RequestExecutor} from '../Http/RequestExecutor';
+import {DocumentConventions} from './Conventions/DocumentConventions';
+
+export class DocumentStore implements IDocumentStore
+{
+  protected Url: string;
+  protected Database: string;
+  protected ApiKey?: string;
+  protected _RequestExecutor: RequestExecutor;
+  protected _Conventions: DocumentConventions; 
+
+  public get RequestExecutor(): RequestExecutor
```

Comment:
Please keep one bracket opening/closing convention - opening brackets on line end and closing ones in new lines.